### PR TITLE
Fix table for llama32 blog

### PR DESCRIPTION
**Pull request by merveenoyan**
@osanseviero

### CODE REVIEW

File: llama32.md [hunk 1]
```diff
@@ -68,37 +68,42 @@ Below you can find some inference examples from the 11B instruction-tuned model
...
   </tr>
   <tr>
     <td style="vertical-align: middle;"><img src="https://huggingface.co/datasets/huggingface/release-assets/resolve/main/weather_events.png" width="300"/></td>
     <td style="vertical-align: middle;">
-      <strong>Where do the droughts happen according to this diagram?</strong><br>
-      <strong>Drought Regions in 2016**</strong>According to the diagram, droughts occurred in two distinct regions in 2016:<br>
+    <strong>Where do the droughts happen according to this diagram?</strong>
+    </td>
+    <td style="vertical-align: middle;">
+      <strong>Drought Regions in 2016</strong>According to the diagram, droughts occurred in two distinct regions in 2016:<br>
```
Comment (osanseviero) [inline llama32.md:87]: Isn't this missing a bullet point, two dots, and space as the other points?

**Review by osanseviero — COMMENTED** [review]
This is nicer! Maybe add a table heading as well

**Comment by merveenoyan** [comment]
@osanseviero addressed
Screenshot 2024-09-27 at 18 03 42
Screenshot 2024-09-27 at 18 03 52

**Review by osanseviero — APPROVED** [review]
Very nice, thanks!

**Pull request merged**